### PR TITLE
Updated Theme Change

### PR DESCRIPTION
**Pull request by spiddubnyy**
Screenshot 2024-03-27 at 12 40 09 AM
Screenshot 2024-03-27 at 12 51 53 AM

### CODE REVIEW

File: client/src/theme/themes.ts [hunk 1]
```diff
@@ -27,6 +27,30 @@ export const themes: { [key: string]: ThemeType } = {
...
+    'bs-body-color': '#FFFFFF', 
+    'bs-btn-primary-bg': '#F5F5F5', 
+    'bs-btn-primary-color': '#000000',
+    'bs-link-color': '#FF0000', 
+    'bs-link-hover-color': '#800000',
+    'bs-background-color': '#FFFFFF',
+    'sidebar-bg-color': '#FFFFFF',
+    'sidebar-text-color': '#000000',
+    'sidebar-text-color-active': '#FF1493'
+  }
```
Comment (StuartGavidia) [inline client/src/theme/themes.ts:54]: Are you able to add color themes for more industries?

Comment (spiddubnyy) [reply client/src/theme/themes.ts:54]: Yes. It is dynamic so if anybody has themes they are interested in, simply add it to the file like the above examples and it should show up.

Comment (spiddubnyy) [reply client/src/theme/themes.ts:54]: I can add more later but paste here if you have any immediate ideas you want before this PR is merged.

File: client/src/App.tsx [hunk 2]
```diff
@@ -80,7 +80,13 @@ function App() {
           <Route path="budget" element={<Budget />} />
           <Route path="calendar" element={<Calendar />}/>
           <Route path="communication" element={<Communication />}/>
-          <Route path="settings" element={<SettingsPage />}/>
+          <Route path="settings" element={
+          <ProtectedRoute>
+            <ThemeProvider>
```
Comment (StuartGavidia) [inline client/src/App.tsx:85]: The ThemeProvider wrapper is already present in the /dashboard path <Route /> component. Lines 83-89 shouldn't be necessary.

File: client/src/theme/ThemeContext.tsx [hunk 3]
```diff
@@ -28,19 +28,29 @@ export const ThemeProvider: React.FC<ThemeProviderProps> = ({ children }) => {
...
   };
 
+  const toggleLightThemeChange = () => {
+    if (themeName == 'light') {
+      applyTheme('light')
+    }
+  } 
+
   useEffect(() => {
+    console.log(themeName)
```
Comment (StuartGavidia) [inline client/src/theme/ThemeContext.tsx:47]: remove console.log()

File: client/src/theme/ThemeContext.tsx [hunk 4]
```diff
@@ -28,19 +28,29 @@ export const ThemeProvider: React.FC<ThemeProviderProps> = ({ children }) => {
   };
 
   const applyTheme = (themeName: string) => {
-    const theme = themes[themeName];
+    const lightTheme = localStorage.getItem('lightTheme') || 'casual';
+    const theme = (themeName == 'light') ? themes[lightTheme.toLowerCase()] : themes['dark']
+    
+    console.log(theme, lightTheme)
```
Comment (StuartGavidia) [inline client/src/theme/ThemeContext.tsx:34]: remove console.log()

File: client/src/App.tsx (hunk 2 above)
Comment (spiddubnyy) [reply client/src/App.tsx]: ok removed

File: client/src/theme/ThemeContext.tsx (hunk 3 above)
Comment (spiddubnyy) [reply client/src/theme/ThemeContext.tsx]: removed

File: client/src/theme/ThemeContext.tsx (hunk 4 above)
Comment (spiddubnyy) [reply client/src/theme/ThemeContext.tsx]: removed

**Review by StuartGavidia — APPROVED** [review]
Great pr!

**Pull request merged**